resources/views/view.blade.php

```diff
@@ -5,8 +5,8 @@
     <link type="text/css" rel="stylesheet" href="{{ Helpers::cacheBusting('dist/main.css') }}">
 
     <link rel="shortcut icon" href="favicon.ico">
-    <link rel="apple-touch-icon" href="Lychee-front/images/apple-touch-icon-iphone.png" sizes="120x120">
-    <link rel="apple-touch-icon" href="Lychee-front/images/apple-touch-icon-ipad.png" sizes="152x152">
+    <link rel="apple-touch-icon" href="img/apple-touch-icon-iphone.png" sizes="120x120">
+    <link rel="apple-touch-icon" href="img/apple-touch-icon-ipad.png" sizes="152x152">
```

Comment:
```suggestion
    <link rel="apple-touch-icon" href="{{ asset('img/apple-touch-icon-iphone.png') }}" sizes="120x120">
    <link rel="apple-touch-icon" href="{{ asset('img/apple-touch-icon-ipad.png') }}" sizes="152x152">
```

I have no experiences with Blade templates and don't fully understand how they work and how what final relative location looks like seen from the perspective of the client, but I assume that we need `asset` here as well to be fully on the save side? Unfortunately, I have no real chance to test it.

Reply:
No, because the files are exposed as they are in the `public` directory. This should be just how it is in the template in the final HTML.

Reply:
OK, but in this case we should not use `asset` for `$photo['thumb']` in

https://github.com/LycheeOrg/Lychee/blob/fba9dcba7b350914fbf843004b1323ff5d62cbcf/resources/views/emails/photos-added.blade.php#L20

either. The URL is also relative to the `public` directory. The docs are not very clear about what `asset` does besides converting the given path into an absolute URL, but at least we should be consistent about its usage.